docs/reference/metadata/optional_metadata.md

```diff
@@ -1,7 +1,5 @@
 # Optional metadata fields list
 
-_Introduced in InvenioRDM v12_
```

Comment:
> I removed the v12 tag since it's not fully accurate (some of the fields will only be in v13)

So do you want to mark this as Introduced in v13 or granularly indicate which were added in v12 and which in v13?

Reply:
I've added the v13 additions next to the new fields. I don't feel like we need to be super explicit about that these fields weren't around in non-supported versions like v11....but could if you think we should.